src/content/docs/tutorial-create-alerts/create-an-alert.mdx

```diff
@@ -13,37 +17,148 @@ import alertsThresholds from 'images/alerts_screenshot-crop_set-thresholds.webp'
+
+4. Select **Next** once you've finished to save your progress and move on to the next step.
+
+    </Side>
+</SideBySide>
+        </TabsPageItem>
+        <TabsPageItem id="3">
+While using charts is a great way to create alert conditions from premade queries, sometimes you need to create them from scratch. The process for creating a new alert condition from a NRQL query is almost the exact same as creating it from a chart: the key difference is that you write the NRQL query manually instead of using a pre-generated one. 
+
+    <img
+    title="NRQL alert"
+    alt="A screenshot displaying an alert condition created with a simple NRQL query"
+    src={alertsNRQL}
+    />
 
-1. Go to any dashboard. Then, find a chart you want to create an alert from, click the **...** menu, and select **Create alert condition**.
+1. From **one.newrelic.com**, select [**Alerts & AI**](https://one.newrelic.com/alerts-ai), then [**Alert conditions (policies)**](https://one.newrelic.com/alerts-ai/alerts-classic/policies).
 
-2. Select the signal, or stream of data, you want to use to create the alert condition. To keep things simple, select the **Build a query** option for your first condition.
+2. Select **New alert condition**.
 
-3. Modify the NRQL query to match what you want you to monitor, if necessary. For now, we recommend leaving the query as is, but once you have a better handle on alerts and NRQL queries, you can change the query to monitor nearly any data threshold.
+3. Write the NRQL query to create the condition. For example, if you wanted to monitor slow transactions, you can enter:
+```sql
+SELECT average(duration) from Transaction facet name
+```
 
 4. Select **Next** once you've finished to save your progress and move on to the next step.
-    </Step>
-    <Step>
+
+<Callout variant="tip">
+NRQL stands for New Relic's Query Language—a standardized query method that powers all our charts, and which you can use to analyze and query your data. If this is your first time working with NRQL, you may want to check out our [NRQL docs](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) to learn more about NRQL, including [how it works](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-how-nrql-works/) and an [introductory tutorial](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial/).
+</Callout>
+        </TabsPageItem>
+        <TabsPageItem id="2">
+<SideBySide>
+    <Side>
+One of the easiest ways to get started is to use our **Golden signal or metric** option. When selected, this capability gives you a guided creation path for making a condition. All you have to do is select the options you want and let us do the rest!
```

Comment:
Could we make this the first tab that users immediately see? If it's the easiest way, then we should probably default it so they don't miss the hack

Reply:
I spoke about this with the SME, we came to the conclusion that charts are the most common way that people make alerts, so we settled on this order. I'd say let's test it out and see and in a couple of months when the UI gets it's revamp we can revisit and make any necessary changes since we'll be making changes to the page anyway.